simple-erc20-spl-hardhat/README.md

```diff
@@ -0,0 +1,111 @@
+# Example deploying ERC20 to Neonlabs devnet using Hardhat
+
+This directory contains all the files necessary to deploy simplest ERC20-like contract using Neon onto the Solana blockchain.
```

Comment:
simple ERC20 Spl contract and link to https://docs.neon-labs.org/docs/developing/deploy_facilities/interacting_with_spl_tokens